app/controllers/search_controller.rb

```diff
@@ -1,6 +1,7 @@
 class SearchController < ApplicationController
+  before_action :require_user
   def index
-    conn = Faraday.get("http://localhost:9292/images/#{params[:search]}")
+    conn = Faraday.get("https://damp-forest-93176.herokuapp.com/images/#{params[:search]}")
```

Comment:
![](https://media3.giphy.com/media/ys8zByv0Muaxa/200.gif?cid=5a38a5a2a8rdb2jkicsrfwdjqeoajpj6yd992q75brcq8m81&rid=200.gif)
"damp forest" live in action!